modules/trusted-firmware-m/CMakeLists.txt

```diff
@@ -187,7 +187,10 @@ if (CONFIG_TFM_PSA_FRAMEWORK_HAS_MM_IOVEC)
     )
 endif()
 
-if (CONFIG_NFCT_PINS_AS_GPIOS)
+# Get the nfct-pins-as-gpios property from the device tree
+dt_prop(nfct_pins_as_gpios_dt PATH "/uicr" PROPERTY "nfct-pins-as-gpios")
+
+if (CONFIG_NFCT_PINS_AS_GPIOS OR ${nfct_pins_as_gpios_dt})
```

Comment:
I don't suppose it would be possible to have CONFIG_NFCT_PINS_AS_GPIOS be true when the DT node is set like that?

That would fix all usage of CONFIG_NFCT_PINS_AS_GPIOS instead of just this one.

Reply:
discussed offline